examples/frameworks/adk_demo/src/nat_adk_demo/register.py

```diff
@@ -0,0 +1,102 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+
+from nat.builder.builder import Builder
+from nat.builder.function_info import FunctionInfo
+from nat.builder.framework_enum import LLMFrameworkEnum
+from nat.cli.register_workflow import register_function
+from nat.data_models.function import FunctionBaseConfig
+
+from . import weather_update_tool  # noqa: F401  # pylint: disable=unused-import
+
+logger = logging.getLogger(__name__)
+
+
+class ADKAIQConfig(FunctionBaseConfig, name="adk"):
```

Comment:
NAT would be superfluous. Since this is a function, maybe use function in the config name?
```suggestion
class ADKFunctionConfig(FunctionBaseConfig, name="adk"):
```